pysat/instruments/icon_euv.py

```diff
@@ -61,28 +83,27 @@ def init(self):
         modified in-place, as desired.
 
     """
-    logger.info("Mission acknowledgements and data restrictions will be printed " +
-          "here when available.")
+    logger.info(mm_icon.ackn_str)
+    self.meta.acknowledgements = mm_icon.ackn_str
+    self.meta.references = ''.join((mm_icon.refs['mission'],
+                                    mm_icon.refs['euv']))
+
     pass
 
 
 def default(inst):
-    """Default routine to be applied when loading data.
+    """Default routine to be applied when loading data. Adjusts epoch timestamps
+    to datetimes and removes variable preambles.
 
     Parameters
     -----------
     inst : (pysat.Instrument)
         Instrument class object
 
-    Note
-    ----
-        Removes ICON preamble on variable names.
-
     """
 
-    import pysat.instruments.icon_ivm as icivm
-    inst.tag = 'level_2'
-    icivm.remove_icon_names(inst, target='ICON_L2_EUV_Daytime_OP_')
+    mm_gen.convert_timestamp_to_datetime(inst, sec_mult=1.0e-3)
+    mm_gen.remove_leading_text(inst, target='ICON_L26_')
```

Comment:
```suggestion
    if not inst.kwargs['keep_original_names']:
        mm_gen.remove_leading_text(inst, target='ICON_L26_')
```

Reply:
Users can change the setting at instantiation,

fuv = pysat.Instrument('icon', 'fuv', keep_original_names=True)

Reply:
should this also be in init?

Reply:
think I figured it out